CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to GenLayer Simulator
+
+We're thrilled that you're interested in working on the GenLayer Simulator! This document will guide you through the contribution process.
+
+## What is the GenLayer Simulator?
+
+The GenLayer Simulator is an interactive sandbox designed for developers to explore the potential of GenLayer's Intelligent Contracts. It replicates the GenLayer network's execution environment and consensus algorithm, providing a controlled and local environment to test different ideas and behaviors.
+
+Begin by setting up your development environment to start contributing: Follow our [README](https://github.com/yeagerai/genlayer-simulator/blob/main/README.md) for detailed setup instructions.
+
+## How You Can Contribute?
+
+Contributions to the GenLayer Simulator are welcome in several forms:
+
+### Testing the Simulator and Providing Feedback
+
+Help us enhance user experience and system functionality.
+- **Feature Testing**: Provide feedback on new features in our [Discord channel](https://discord.gg/5Gq4nCxe).
+- **Report Bugs**: Submit bugs through our [GitHub issues](https://github.com/yeagerai/genlayer-simulator/issues).
+
+### Sharing New Ideas and Use Cases
+
+Innovate with us by proposing new use cases for Intelligent Contracts.
+- **Idea Submission**: Share your innovative ideas in our [Discord channel](https://discord.gg/5Gq4nCxe).
+
+### Writing Code
+
+Contribute to feature development or bug fixes.
+- **How to Start**: Check our [issue tracker](https://github.com/yeagerai/genlayer-simulator/issues) for tasks labeled `help wanted`.
+
+### Improving Documentation
+
+Enhance our documentation to help users and developers get the most out of the Simulator.
+- **Start Here**: Review our [Documentation Repository](https://github.com/yeagerai/genlayer-docs).
+
+## Review Process
+
+We strive to maintain high-quality code and ensure that all contributions align with our goals. Here’s our process:
+
+- **Pull Request**: Submit your changes through a pull request (PR).
+- **Automated Tests**: Your PR will automatically be tested. Ensure all tests pass to proceed.
```

Comment:
@jmlago @cristiam86 do we have any tests at the moment?

We should be asking people to provide tests with their feature submissions, but that's a little tricky if we don't have a framework set up

Reply:
Not yet, we have to add tests, coverage etc.

We don't have it yet because most PRs contain breaking changes and that means redoing tests. But once we have a first stable version I think is a good moment to start adding tests.

Reply:
Since it will be setup we can leave this info here